### PR TITLE
OTT-1195 :: Always update fetchStatus with dynamic fetch status

### PR DESCRIPTION
**Pull request by pm-nikhil-vaidya**
# Description

Please add change description or link to ticket, docs, etc.

# Checklist:

- [ ] PR commit list is unique (rebase/pull with the origin branch to keep master clean).
- [ ] JIRA number is added in the PR title and the commit message.
- [ ] Updated the `header-bidding` repo with appropiate commit id.
- [ ] Documented the new changes.

For Prebid upgrade, refer: https://inside.pubmatic.com:8443/confluence/display/Products/Prebid-server+upgrade

### CODE REVIEW

**Review by pm-jaydeep-mohite — DISMISSED** [review]
LGTM

File: floors/floors.go [hunk 1]
```diff
@@ -153,15 +153,19 @@ func resolveFloors(account config.Account, bidRequestWrapper *openrtb_ext.Reques
 		account.PriceFloors.Fetch.URL = reqFloor.Location.URL
 	}
 	account.PriceFloors.Fetch.AccountID = account.ID
-	fetchResult, fetchStatus := priceFloorFetcher.Fetch(account.PriceFloors)
+	var fetchResult *openrtb_ext.PriceFloorRules
+	var fetchStatus = openrtb_ext.FetchNone
```
Comment (Pubmatic-Dhruv-Sonone) [inline floors/floors.go:157]: This can be a single var block

File: floors/floors.go [hunk 2]
```diff
@@ -153,15 +153,19 @@ func resolveFloors(account config.Account, bidRequestWrapper *openrtb_ext.Reques
...
+	if shouldUseDynamicFetchedFloor(account) {
```
Comment (ashishshinde-pubm) [inline floors/floors.go:158]: are we making any behaviour change here ? 
previous code was calling priceFloorFetcher.Fetch without checking shouldUseDynamicFetchedFloor.

Comment (pm-nikhil-vaidya) [reply floors/floors.go:161]: Here we do not change the functionality
We just avoided extra function call when dynamic data is disabled

File: floors/floors.go (hunk 1 above)
Comment (pm-nikhil-vaidya) [reply floors/floors.go]: Done

**Review by pm-jaydeep-mohite — APPROVED** [review]
LGTM

**Review by pm-nilesh-chate — APPROVED** [review]

**Pull request merged**